Code/Database/AppveyorScriptDatabase.bat

```diff
@@ -13,3 +13,5 @@ SET PASSWORD="Password12!"
 SET INPUT=%cd%\SQLCreateQuery.sql
 
 %SQLCMD% -S%SERVER% -d%DB% -U%LOGIN% -P%PASSWORD% -i%INPUT% -b
+SET INPUT=%cd%\InsertScript.sql
+%SQLCMD% -S%SERVER% -d%DB% -U%LOGIN% -P%PASSWORD% -i%INPUT% -b
```

Comment:
Search in UI doesnt update when new type of material selected (i think it should, but then again UI not our focus so could be ok)